src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      acc[property] = value;
+    }
+
+    return acc;
+  }, {});
 }
 
 module.exports = convertToObject;
```

Comment:
It's a good practice to provide a named export instead of a default export if you plan to export multiple functions or variables from this module in the future. Consider using `exports.convertToObject = convertToObject;`.